docs/upcoming_changes/9417.new_feature.rst

```diff
@@ -0,0 +1,4 @@
+Support math.log2
```

Comment:
Shouldn't this be a `CUDA` change not a `new_feature`? Also, the title has `math.log2` and the content has `np.log2`, I think the title is incorrect?